examples/server/server.cpp

```diff
@@ -613,7 +613,7 @@ struct server_context {
 
     gpt_params params;
 
-    llama_batch batch;
+    llama_batch batch = {};
```

Comment:
This also fixes a crash in the server when loading a model fails and `llama_batch_free` is called on an uninitialized batch.